src/components/accessories/billTable/styles.scss

```diff
@@ -39,13 +39,13 @@
     }
     .filterBillForm__item {
       //background-color: $c-grey-lighter;
-      padding: 0px 10px;
-      width: 33.33%;
+      padding: 0px 3px;
+      width: 14.28%;
       @include susy-media($narrow) {
-        padding: 0px 10px;
+        width: 50%;
       }
       @include susy-media($tablet_land) {
-        padding: 0px 10px;
+        width: 50%;
```

Comment:
We're not gonna use static width/height for display inline element but we use the flex property.

Reply:
Plus, in small desktop, the filter's items content are hidden by an overflow. I suggest to rethink the alignament of the fields (maybe you could add some flex-column and then scale down at each breakpoint, this is just a suggestion).

![image](https://user-images.githubusercontent.com/8697550/143605525-97f3bb38-3b30-4754-b590-cb52ea9f8dd0.png)

Reply:
It's pretty close to be OK, there is still something to do about the alignment. When you reduce the browser's window the fields' alignament is pretty funny. 

Proposal solutions, group the fields by:
4 date fields
2 other fields
1 button

one type for each row.

Reply:
Ok thanks for the suggestion